team-builder/src/App.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import './App.css';
+import TeamMembers from './components/TeamMembers';
+import TeamMemberForm from './components/Form';
+
+import { Jumbotron } from 'reactstrap';
+
+function App() {
+  const [ teamList, setTeamList ] = useState([
+    {
```

Comment:
This is wonderful, I just thought I'd suggest creating this outside of the file into a dummyData section.  But that's my personal opinion.